recipe/meta.yaml

```diff
@@ -55,6 +60,12 @@ test:
     - cmake --build test-linking-bld
     - cmake --build test-linking-bld --target test
 
+    # Test RTTI fix for macOS (exception handling across shared library boundaries)
+    # This test verifies that RequestFailedException can be caught correctly,
+    # addressing the issue described in:
+    # https://github.com/conda-forge/azure-storage-blobs-cpp-feedstock/issues/22
+    - bash test_rtti_macos.sh  # [osx]
```

Comment:
Unfortunately this will only run on osx-64 and not osx-arm64 as in the original bug report

Reply:
Why?

Reply:
Because the test section isn't run when the software is cross-compiled:

https://github.com/conda-forge/azure-core-cpp-feedstock/blob/bfdf9f804871fe4524f670e714a96b41e41e476c/conda-forge.yml#L7-L8

My understanding is that you can compile a osx-arm64 binary on osx-64 architecture, but you can't execute it

Reply:
What if one runs it natively?